.github/workflows/cli.yml

```diff
@@ -0,0 +1,64 @@
+name: Run Tests
+
+on:
+  pull_request:
+    branches:
+      - master
+      - v1
+
+env:
+  GO111MODULE: on
+  GOPROXY: https://proxy.golang.org
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        go: [1.11, 1.12, 1.13]
+    name: ${{ matrix.os }} @ Go ${{ matrix.go }}
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Set up Go ${{ matrix.go }}
+        uses: actions/setup-go@v1
+        with:
+          go-version: ${{ matrix.go }}
+
+      - name: Set GOPATH and PATH
+        run: |
+          echo "##[set-env name=GOPATH;]$(dirname $GITHUB_WORKSPACE)"
+          echo "##[add-path]$(dirname $GITHUB_WORKSPACE)/bin"
+        shell: bash
+
+      - name: Checkout Code
+        uses: actions/checkout@v1
+        with:
+          ref: ${{ github.ref }}
+
+      - name: Install Dependencies
+        run: |
+          mkdir -p $GOPATH/bin
+          curl -s https://api.github.com/repos/urfave/gfmrun/releases/latest | grep gfmrun-linux-amd64 | grep download | cut -d '"' -f 4 | wget -O $GOPATH/bin/gfmrun -i -
+          npm install markdown-toc
+
+      - name: Run Tests (v1)
+        if: contains(github.ref, 'v1')
```

Comment:
Looks like the current setup doesn't actually ever execute the tests. Note the gray squares with `0s`:
<img width="1099" alt="image" src="https://user-images.githubusercontent.com/5321575/70293721-2d015880-17af-11ea-967b-efe542ae9399.png">


I played around with this a bit on my fork, it looks like `github.ref` refers to the branch triggering the PR, and `github.base_ref` is the target. So if we're discriminating based on where the PR is supposed to land, we'll want to use `github.base_ref` to get tests to run

Source: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/contexts-and-expression-syntax-for-github-actions#github-context

Reply:
@rliebz I noticed this now. Will fix.